warehouse/macros/define_gtfs_guidelines.sql

```diff
@@ -1,4 +1,29 @@
+--
+-- ENTITIES
+--
+{% macro schedule_feed() -%}
+'schedule_feed'
+{%- endmacro %}
+
+{% macro schedule_url() -%}
+'schedule_url'
+{%- endmacro %}
+
+{% macro rt_feed() -%}
+'rt_feed'
+{%- endmacro %}
+
+{% macro rt_url() -%}
+'rt_url'
+{%- endmacro %}
+
+{% macro organization() -%}
+'organization'
+{%- endmacro %}
+
```

Comment:
Between you and @owades what is the plan for adding "service"? Wondering if @owades needs to do that on #2206 or not

Reply:
I'll add the macros here; then @owades just has to use them in intended_checks